app/views/templates/payments/PaymentsHistoryTabsContent.scala.html

```diff
@@ -17,83 +17,67 @@
-    }
-
-    @if(showPreviousPaymentsTab) {
-      | <a href="#nonJS-previous-payments">@messages("paymentsHistory.previousPayments.heading")</a>
-    }
-  </p>
-}
+  showPreviousPaymentsTab: Boolean
+  )(implicit messages: Messages, appConfig: AppConfig, user: User)
 
 @for(year <- years) {
-  <section @if(javascriptEnabled){id="@year" class="tabcontent" role="tabpanel"} else {id="nonJS-@year"}>
-    <h2 class="heading-medium">@year</h2>
+  <div id="year-@year" class="govuk-tabs__panel@if(year != years.head) { govuk-tabs__panel--hidden}">
+    <h2 class="govuk-heading-m">@year</h2>
     @if(transactions.exists(_.clearedDate.getOrElse("").toString.contains(year.toString))) {
-      <table>
-        <thead>
-          <tr>
-            <th scope="col">
-              <div class="visuallyhidden">@messages("paymentsHistory.paymentReceivedTitle")</div>
-            </th>
+    <table class="govuk-table">
+      <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th class="govuk-table__header govuk-visually-hidden" scope="col">
+            @messages("paymentsHistory.paymentReceivedTitle")
+          </th>
 
-            <th scope="col">
-              <div class="visuallyhidden">@messages("paymentsHistory.descriptionTitle")</div>
-            </th>
+          <th class="govuk-table__header govuk-visually-hidden" scope="col">
+            @messages("paymentsHistory.descriptionTitle")
+          </th>
+
+          <th class="numeric govuk-table__header govuk-visually-hidden" scope="col">
+            @messages("paymentsHistory.amountTitle")
+          </th>
+        </tr>
+      </thead>
+
+      <tbody class="govuk-table__body">
+        @for(transaction <- transactions.filter(_.clearedDate.getOrElse("").toString.contains(year.toString))) {
+          @paymentsHistoryCharge(transaction)
+        }
+      </tbody>
+    </table>
 
-            <th class="numeric" scope="col">
-              <div class="visuallyhidden">@messages("paymentsHistory.amountTitle")</div>
-            </th>
-          </tr>
-        </thead>
-        <tbody>
-          @for(transaction <- transactions.filter(_.clearedDate.getOrElse("").toString.contains(year.toString))) {
-            @paymentsHistoryCharge(transaction)
-          }
-        </tbody>
-      </table>
     } else {
-      <p>@messages("paymentsHistory.noCurrentYearHistory")</p>
+      <p class="govuk-body">@messages("paymentsHistory.noCurrentYearHistory")</p>
     }
-  </section>
+    </div>
 }
 
 @if(showPreviousPaymentsTab) {
-
-  <section
-      @if(javascriptEnabled){
-          id="previous-payments" class="tabcontent" role="tabpanel"
-      }else{
-          id="nonJS-previous-payments"
-      }
-    >
-    <h2 class="heading-medium">@messages("paymentsHistory.previousPayments.heading")</h2>
-    <p>
+  <div id="previous-payments" class="govuk-tabs__panel">
+    <h2 class="govuk-tabs__title">@messages("paymentsHistory.previousPayments.heading")</h2>
```

Comment:
This heading just needs to be set to `govuk-heading-m`. Sorry if my last comment about this was confusing, I tagged the wrong line. Basically the issue was that the text in the tabs themselves needed the "tabs title" class, but the actual h2s inside the tab content itself was fine.

The text in the tabs is now currently fine so you just need to change this one class, as right now this h2 is not displaying.